core/src/skybox.cpp

```diff
@@ -0,0 +1,166 @@
+#include "knoting/skybox.h"
+
+namespace knot {
+namespace components {
+
+struct PosColorTexCoord0Vertex {
+    float m_x;
+    float m_y;
+    float m_z;
+    uint32_t m_rgba;
+    float m_u;
+    float m_v;
```

Comment:
Can this be a `vec2`?

Reply:
vec2 pos vertex class is obsolete and is handled by instanceMesh